automation/src/test/scala/GoogleSpec.scala

```diff
@@ -0,0 +1,26 @@
+import org.broadinstitute.dsde.firecloud.test.WebBrowserSpec
+import org.scalatest.ParallelTestExecution
+import org.scalatest.selenium.WebBrowser
+import org.scalatest.FlatSpec
+
+/**
+  * Example test for running with Selenium Grid until we work out docker
+  * networking to be able to access FiaBitC.
+  */
+class GoogleSpec extends FlatSpec with WebBrowserSpec with ParallelTestExecution with WebBrowser {
+
+  behavior of "Google"
+
+  it should "have a search field" in withWebDriver { implicit driver =>
+    go to new GooglePage()
+    assert(find("q").isDefined)
```

Comment:
Looks like `ShouldMatchers` is deprecated in favor of `Matchers` which we are using in some of the other tests.

Thanks for the tip on using `defined` with matchers! I wasn't sure how to do that and I'm pretty sure there are some places where I can put that to good use.